wercker.yml

```diff
@@ -5,11 +5,14 @@ services:
 build:
     steps:
         - virtualenv:
-            install_wheel: false
+            install_wheel: true
         - script:
             name: Install remodel
             code: pip install .
         - script:
+            name: Install python-dateutil (for six)
```

Comment:
I had some pain to get passing on wercker.  Actually, six would not install directly with pip, so after half dozen attempts I tried the python-dateutil, which pulls in six.  I also changed your setup.py and I actually think that is what got it working, not the change to .yml